tests/conftest.py

```diff
@@ -58,6 +58,15 @@ def app_config(app_config):
         "R": "Remote",
     }
     app_config["FILES_REST_DEFAULT_STORAGE_CLASS"] = "L"
+    app_config["COMMUNITIES_IDENTITIES_CACHE_TIME"] = 60 * 60 * 24
```

Comment:
Maybe we can test the expires (e.g. small timeouts)

Reply:
Right! Good point! thanks :)